create-publication-from-scopus/src/main/java/no/sikt/nva/scopus/ScopusConverter.java

```diff
@@ -61,9 +68,42 @@ private EntityDescription generateEntityDescription() {
         entityDescription.setReference(generateReference());
         entityDescription.setMainTitle(extractMainTitle());
         entityDescription.setContributors(generateContributors());
+        entityDescription.setTags(generatePlainTextTags());
         return entityDescription;
     }
 
+    private List<String> generatePlainTextTags() {
+        var authorKeywordsTp = extractAuthorKeyWords();
+        return Objects.nonNull(authorKeywordsTp)
+                   ? authorKeywordsTp
+            .getAuthorKeyword()
+            .stream()
+            .map(something -> something.getContent()
+                .stream()
+                .map(this::extractContentString)
+                .collect(Collectors.joining()))
+            .collect(Collectors.toList())
+                   : null;
+    }
+
+    private String extractContentString(Object content) {
+        if (content instanceof String) {
```

Comment:
Honestly, I'd just return the XML…I have no clue what to do with the data — if they have so specific things, I guess that they should use a controlled vocabulary

Reply:
The XML version has already been extracted in previous PR. 
When I look at the generated classes it seems the valid root element is either String, JAXBelement<"SupTp"> og JAXBelement<"InfTp">.
The InfTp and SupTp both contain an ArrayList of string or JAXBelement<"suptp"> JAXBelement<"InfTp">

Making the frontend parse raw xml inside a json, will only make this piece of code get moved to the frontend instead of the backend. 

(The "" are to prevent github comment from parsing my <"tags"> from the input. )